lib/elixir/lib/enum.ex

```diff
@@ -803,11 +803,13 @@ defmodule Enum do
   @spec filter_map(t, (element -> as_boolean(term)),
     (element -> element)) :: list
 
-  def filter_map(enumerable, filter, mapper) when is_list(enumerable) do
+  def filter_map(enumerable, filter, mapper)
+  when is_list(enumerable) and is_function(filter, 1) and is_function(mapper, 1) do
```

Comment:
Heresy! 😄 I fought long and hard to get that in @lexmag's style guide 😛 

Jokes aside, the reason we went with this is that then you create these whitespace holes (think of how long `defmacrop` is 😕) that are not consistently big, depending on the thing we're indenting to :(